### ISSUE TITLE
Is basic roomscale a possibility?

### ISSUE THREAD

**Issue by AerowynX** [Nibre/MotherVR#44]
I been playing standing which is way more immersive than sitting but you need to constantly reset your view everytime you interact with anything that takes camera control so any terminal or save point or ladder and such or else it often has you facing the wrong direction. Also when turning around constantly the arms start to get out of position so you gotta recenter. Just curious if basic 360 standing will ever be a possibility or even better yet full roomscale so i can hide in the corner in my room while hiding from the aliens :)

**Comment by Aesop112** [comment]
Id heard from dev that motion controls were also possible.

**Comment by AerowynX** [comment]
Sounded like just basic mapping of gamepad controls like VorpX does..not
actual interaction..don't think that is possible without having more access
to games code

**Comment by Aesop112** [comment]
He shouldn’t have access to the models and animations, as per this guide: https://steamcommunity.com/sharedfiles/filedetails/?id=423489181

So I don’t see why not.

**Comment by AerowynX** [comment]
This game would be bonkers with full roomscale and full motion controller
implementation..hiding in the corner of my room peaking around the corner
as the alien paces around.. would be nuts

**Comment by Aesop112** [comment]
Yeah, that’s the dream. For peaking, I would assume you would need to keep your controllers in the original hiding spot while you move your head.

**Comment by MattFiler** [comment]
Mod guide writer here, models can be exported but not re-imported. I could mod the game to make the motion tracker easier to see in VR if people were interested.

**Comment by Aesop112** [comment]
I would love that, but I’d love it more if you could attach it to the left controller more lol. The big problem right now is virtually all text is rediculously tiny and totally unreadable. I’m sure the modder of this vr port would welcome any help from individuals familiar with modding this game.
Thank you, and you can contact the author via github. Like I said I’m sure he’d be happy to get some help. Have a good one!

**Comment by MattFiler** [comment]
I tried contacting Nibre to help out a week or so ago but he hasn't replied yet. Would be happy to do UI fixes, etc.

**Comment by AerowynX** [comment]
Awesome man I'll hold off playing then:)

**Comment by AerowynX** [comment]
any progress on roomscale? it's really only thing holding me back from enjoying this as every time the camera control is taken it doesn't know my position so things get all messed up

**Comment by NeonUniverse** [comment]
Roomscale keep in mind would take a bit to implement, it isn't just a simple line of code. Getting to this point alone is a great achievement as it stands, VR and all. Give it some time and soon small roomscale will be a thing, but remember to be patient.
